src/test/indexTest.ts

```diff
@@ -118,6 +118,24 @@ describe('index', () => {
                 done();
             });
         }).timeout(5000);
+
+        it('parameters of the CommaDelimitedList type should accept lists as values', (done) => {
+            exec('node lib/index.js validate testData/valid/yaml/parameters_type_commadelimitedlist.yaml --parameters SomeParam="test\\,dev\\,prod"', function(error, stdout, stderr) {
```

Comment:
Can this be mentioned in the docs?

something\,something\,something is possible